balancer/weightedroundrobin/config.go

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package weightedroundrobin
+
+import (
+	"time"
+
+	"google.golang.org/grpc/serviceconfig"
+)
+
+type lbConfig struct {
+	serviceconfig.LoadBalancingConfig `json:"-"`
+
+	EnableOOBLoadReport     bool          `json:"enableOobLoadReport,omitempty"`
+	OOBReportingPeriod      time.Duration `json:"oobReportingPeriod,omitempty"`
+	BlackoutPeriod          time.Duration `json:"blackoutPeriod,omitempty"`
+	WeightExpirationPeriod  time.Duration `json:"weightExpirationPeriod,omitempty"`
+	WeightUpdatePeriod      time.Duration `json:"weightUpdatePeriod,omitempty"`
+	ErrorUtilizationPenalty float64       `json:"errorUtilizationPenalty,omitempty"`
```

Comment:
Ummmmm I don't know if I agree with deferring to a linked url for explanation. I feel like we should have comments for this in the codebase? Similar to how we do in xDS update structs internal (I know we convert from proto and it's required since their exported), but it's much easier to understand code for a reader that way.

Reply:
Copied the comments from the gRFC here.  In the process I noticed that I missed the default 1.0 error utilization penalty, so that was a good bug to catch.